### PR TITLE
[BW] Fix UIKit code examples in docusaurus

### PR DESCRIPTION
**Pull request by alteral**

### CODE REVIEW

File: docusaurus/docs/iOS/basics/logs.md [hunk 1]
```diff
@@ -31,8 +31,7 @@ By default, the logs will provide basic text to your console. Still, in the SDK,
 
 ```swift
 LogConfig.formatters = [
-    PrefixLogFormatter(prefixes: [.info: "ℹ️", .debug: "🛠", .warning: "⚠️", .error: "🚨"]),
-    PingPongEmojiFormatter()
+    PrefixLogFormatter(prefixes: [.info: "ℹ️", .debug: "🛠", .warning: "⚠️", .error: "🚨"])
 ]
 ```
```
Comment (alteral) [inline docusaurus/docs/iOS/basics/logs.md:36]: We do not have an example of `PingPongEmojiFormatter` class. I think it's better to remove this line or copy-paste a code example from `StreamChatSample` target.

File: docusaurus/docs/iOS/uikit/custom-components.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ class iMessageChannelListItemView: ChatChannelListItemView {
 
     override func setUpAppearance() {
         super.setUpAppearance()
-`
+
         customUnreadView.backgroundColor = tintColor
         customUnreadView.layer.masksToBounds = true
         customUnreadView.layer.cornerRadius = 5
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/uikit/custom-components.md:159]: Most likely this is also wrong, cornerRadius should not be applied in `setUpAppearance` :/

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅ Just one minor NIT :)

File: docusaurus/docs/iOS/uikit/custom-components.md (hunk 2 above)
Comment (alteral) [reply docusaurus/docs/iOS/uikit/custom-components.md:159]: @nuno-vieira, it comes from [here](https://github.com/GetStream/stream-chat-swift/blob/develop/Examples/iMessageClone/iMessageChatChannelListItemView.swift#L16). By the way, I don't think we need `cornerRadius` on iMessageClone, do we?

Comment (alteral) [reply docusaurus/docs/iOS/uikit/custom-components.md]: Updated

Comment (alteral) [reply docusaurus/docs/iOS/uikit/custom-components.md]: Oh, just checked it across all docs, actually, it is everywhere. Let me commit one more time to fix it there.

Comment (alteral) [reply docusaurus/docs/iOS/uikit/custom-components.md]: @nuno-vieira, done, check it out please 👍

**Pull request merged**